### ISSUE TITLE
Abnormal results during the supervised training stage

### ISSUE THREAD

**Issue by ZhimaoPeng** [kleinzcy/Cr-KD-NCD#5]
Thanks for your work and code, this is very helpful to us !

When supervised pre-training was performed on the aircraft and CUB data sets as directed, the pretrain acc was very low, resulting in poor class discovery results in the second stage.

This is the pre-training accuracy on the CUB dataset:
![CUB_pre_train](https://github.com/kleinzcy/Cr-KD-NCD/assets/37007544/c62477ea-05e4-4b58-a04a-78b63a196076)

This is the pre-training accuracy on the aircraft  dataset:
![aircraft_pre_train](https://github.com/kleinzcy/Cr-KD-NCD/assets/37007544/5f3d285b-d494-4295-b530-68dcbc98d2dd)

But with the pre-trained model provided by the code base, the class discovery results are normal, is there any way to get the correct pre-trained model? thanks!

**Comment by kleinzcy** [comment]
Thanks for your attention. 

Given the limited information, it is hard to assess where the problem is. Maybe, you can provide the whole log.

We also provide the download link of pretrained model in the readme.

**Comment by ZhimaoPeng** [comment]
> Thanks for your attention.
> 
> Given the limited information, it is hard to assess where the problem is. Maybe, you can provide the whole log.
> 
> We also provide the download link of pretrained model in the readme.

All operations are performed according to the readme, here are the log files  running on both datasets:

[pretrain_cub.txt](https://github.com/kleinzcy/Cr-KD-NCD/files/14946436/pretrain_cub.txt)




[pretrain_aircraft.txt](https://github.com/kleinzcy/Cr-KD-NCD/files/14946440/pretrain_aircraft.txt)
Is it possible that the hyperparameters in the configuration file are incorrect?

**Comment by kleinzcy** [comment]
Maybe the leraning rate in pretrain is too low. The local code is not available for me now. You can change the lr to 1e-1 and try again.

**Comment by ZhimaoPeng** [comment]
> Maybe the leraning rate in pretrain is too low. The local code is not available for me now. You can change the lr to 1e-1 and try again.

The problem has been solved, thanks!
